src/pages/Download.js

```diff
@@ -135,14 +135,14 @@ const Download = () => {
               <TabPanel>
                 <TabSection>
                   <div>
-                    <SectionTitle>Linux x64</SectionTitle>
+                    <SectionTitle>Linux (x86/x64)</SectionTitle>
```

Comment:
This title should be Linux (x64) only and below "Download UPBGE 0.30.0 (64-bit)" only too. Really when in the build name we include x86_64 is a form to say x64. Basically, x86_64, x64 and amd64 are ways of saying "64-bit".

Apart from this, i'm ok with this change. Great work.